go.mod

```diff
@@ -0,0 +1,78 @@
+module github.com/siderolabs/go-talos-support
+
+go 1.22.1
+
+require (
+	github.com/containerd/containerd v1.7.13
+	github.com/cosi-project/runtime v0.4.0-alpha.9
+	github.com/dustin/go-humanize v1.0.1
+	github.com/hashicorp/go-multierror v1.1.1
+	github.com/siderolabs/talos/pkg/machinery v1.6.0-alpha.2.0.20240307132915-3c8f51d707b8
```

Comment:
this looks like a strange tag?

Reply:
That's `github.com/siderolabs/talos/pkg/machinery main` then `go mod tidy`. This lib needs the latest main as it uses extracted modules

Reply:
we have 1.7.0-alpha.1 tagged at least, so it can't be this tag

Reply:
I wonder why it doesn't pick it up automatically 🤔

Reply:
I see only alpha.0 btw, which doesn't have my changes.

Reply:
ok, still should be 1.7 :)